src/components/EditModal.vue

```diff
@@ -82,16 +82,28 @@ export default {
     }
   },
   data() {
-    return {
-      title: this.todo.title,
-      priority: this.todo.priority,
-      category: this.todo.category,
-      limit: this.todo.limit
-    };
+    // return {
+    //   title: this.todo.title,
+    //   priority: this.todo.priority,
+    //   category: this.todo.category,
+    //   limit: this.todo.limit
+    // };
+    // does this work?
+    return this.todo;
+    // otherwise, (shallow-)clone the object?
+    // return {...this.todo}
+    // if the object has nested object inside, you'd have to deep clone:
+    // import {cloneDeep} from 'lodash'
+    // return cloneDeep(this.todo)
+    // if you don't care about performance you can hack a cloneDeep together real quick:
+    // return JSON.parse(JSON.stringify(this.todo))
   },
   watch: {
+    // 😲😲😲
     // eslint-disable-next-line no-unused-vars
     todo(newVal, oldVal) {
+      // looks a little repetitive
+      // I wonder if there's a way to be easier on your fingers?
```

Comment:
I had the watch because it wouldn't update the edit modal values without it after pressing sort.